lib/canvas/live_events.rb

```diff
@@ -101,6 +101,23 @@ def self.get_submission_data(submission)
     }
   end
 
+  def self.get_attachment_data(attachment)
+    {
+      attachment_id: attachment.global_id,
+      user_id: attachment.global_user_id,
+      display_name: LiveEvents.truncate(attachment.display_name),
+      filename: LiveEvents.truncate(attachment.filename),
+      context_type: attachment.context_type,
+      context_id: attachment.context_id,
+      content_type: attachment.content_type,
+      folder_id: attachment.folder_id,
```

Comment:
I think you're correct about submissions being on `User` context, but I don't know for sure that `'Course'` files are always exposed in the course files tool. I *am* pretty sure that anything with `context_type = 'Course'` and a non-null `folder_id` will be part of the course files though.